src/main/java/mods/railcraft/common/util/misc/EntityIDs.java

```diff
@@ -35,6 +35,7 @@
     public static final int CART_LOCO_STEAM_SOLID = 30;
     public static final int CART_LOCO_STEAM_MAGIC = 31;
     public static final int CART_LOCO_ELECTRIC = 32;
+    public static final int CART_COMMAND_BLOCK = 50;
```

Comment:
I'm not sure that IDs can be changed between versions without breaking save files. Otherwise I would be using an enum (I'm the enum king after all).